ci/nightly/pipeline.yml

```diff
@@ -7,115 +7,151 @@
-        composition: upgrade
-
-- id: limits
-  label: "Product limits test"
-  plugins:
-    - ./ci/plugins/mzcompose:
-        composition: limits
-  timeout_in_minutes: 50
-
-- id: cluster-testdrive
-  label: "Full testdrive against Cluster"
-  plugins:
-    - ./ci/plugins/mzcompose:
-        composition: cluster
-        run: nightly
-  timeout_in_minutes: 30
-
-- id: proxy
-  label: ":squid: proxy"
-  plugins:
-    - ./ci/plugins/scratch-aws-access: ~
-    - ./ci/plugins/mzcompose:
-        composition: proxy
-        args: [--aws-region=us-east-2]
-
-- id: testdrive_workers_1
-  label: ":racing_car: testdrive with --workers 1"
-  timeout_in_minutes: 30
-  plugins:
-    - ./ci/plugins/scratch-aws-access: ~
-    - ./ci/plugins/mzcompose:
-        composition: testdrive
-        args: [--aws-region=us-east-2, --workers=1]
-
-- id: persistence_testdrive
-  label: ":racing_car: testdrive with --persistent-user-tables"
-  timeout_in_minutes: 30
-  plugins:
-    - ./ci/plugins/scratch-aws-access: ~
-    - ./ci/plugins/mzcompose:
-        composition: testdrive
-        args: [--aws-region=us-east-2, --persistent-user-tables]
-
-- id: feature-benchmark
-  label: "Feature benchmark against latest release"
-  timeout_in_minutes: 120
-  plugins:
-    - ./ci/plugins/mzcompose:
-        composition: feature-benchmark
-        args:
-          - --other-tag
-          - latest
-
-- id: feature-benchmark-persistence
-  label: "Feature benchmark against latest release with persistence"
-  timeout_in_minutes: 120
-  plugins:
-    - ./ci/plugins/mzcompose:
-        composition: feature-benchmark
-        args:
-          - --other-tag
-          - latest
-          - --this-options
-          - "--persistent-user-tables --persistent-kafka-sources"
-          - --other-options
-          - "--persistent-user-tables --persistent-kafka-sources"
-
-- id: aws-config
-  label: "AWS credentials and role assumption"
-  timeout_in_minutes: 30
-  plugins:
-    - ./ci/plugins/scratch-aws-access: ~
-    - ./ci/plugins/mzcompose:
-        composition: aws-config
+steps:
+  - block: Nightly test selection
+    prompt: |
+      What tests would you like to run? As a convenience, leaving all tests
+      unchecked will run all tests.
+    blocked_state: running
+    fields:
+      - select: Tests
+        key: tests
+        options:
+          - { value: coverage }
+          - { value: kafka-matrix }
+          - { value: kafka-multi-broker }
+          - { value: redpanda-testdrive }
+          - { value: redpanda-testdrive-aarch64 }
+          - { value: upgrade }
+          - { value: limits }
+          - { value: cluster-testdrive }
+          - { value: proxy }
+          - { value: testdrive-workers-1 }
+          - { value: persistence-testdrive }
+          - { value: feature-benchmark }
+          - { value: feature-benchmark-persistence }
+          - { value: aws-config }
+        multiple: true
+        required: false
+    if: build.source == "ui"
```

Comment:
So your new TV show is called "Yaml is the new bash" :-) The conditional is at the end of the block it references ...